src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx

```diff
@@ -13,12 +26,65 @@ import {
 
 import './CreateVMFooter.scss';
 
-const CreateVMFooter: FC = () => {
+type CreateVMFooterProps = {
+  vm: V1VirtualMachine;
+  onCancel: () => void;
+  bootableVolumeSelected: V1beta1DataSource;
+};
+
+const CreateVMFooter: FC<CreateVMFooterProps> = ({ vm, onCancel, bootableVolumeSelected }) => {
   const { t } = useKubevirtTranslation();
+  const history = useHistory();
   const [startVM, setStartVM] = useState(true);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [error, setError] = useState<Error | any>(null);
+
+  const handleSubmit = async () => {
+    setIsSubmitting(true);
+    setError(null);
+
+    const pvc = await k8sGet<V1alpha1PersistentVolumeClaim>({
+      model: PersistentVolumeClaimModel,
+      name: bootableVolumeSelected?.spec?.source?.pvc?.name,
```

Comment:
At some point we should create selectors for these.

Reply:
I changed it a bit, while searching for an existing selector, I found getPVC function that is doing exactly same as this code,
about the selectors, I think we tried to avoid creating selectors as there are simply a lot of selectors that can be created